src/pages/ManageBooking.tsx

```diff
@@ -0,0 +1,63 @@
+import { useNavigate } from 'react-router-dom'
+import { ChevronBackIcon } from '../assets'
+import { useBookings, useCarTypes } from '../hooks'
+import { useAuth } from '../context/AuthContext'
+import ManageBookingCard from '../components/cars/ManageBookingCard'
+import LoaderComponent from '../components/ui/Loader'
```

Comment:
We can call it the Loader only too
```suggestion
import Loader from '../components/ui/Loader'
```